docs/kb/compaction.rst

```diff
@@ -188,7 +188,31 @@ Which Strategy is best to use
 -----------------------------
 Use the table in :ref:`Which strategy is best <which-strategy-is-best>` to determine the right strategy for your needs.
 
+.. _compaction-off-strategy-compaction:
 
+Off-strategy Compaction
+----------------------------
+
+Off-strategy compaction is an internal mechanism used by ScyllaDB to improve the efficiency of node operations, 
+such as ``bootstrap``, ``repair``, or ``replace``. It keeps SStables created by node operations 
+in a separate “maintenance” data set ignored by the regular compaction (STCS, LCS, etc.).
+
+After the node operation is completed, all the new SSTables kept in the maintenance set are 
+reshaped and then integrated into the main set of SStables to fall under the regular compaction strategy. 
+
```

Comment:
One correction:

After a table has been processed during node operation, all of its new sstables kept in maintenance set ...

Please reword it as you wish.